.ci/wercker.sh

```diff
@@ -98,7 +98,7 @@ no-error-xwiki)
   CS_POM_VERSION=$(mvn -e -q -Dexec.executable='echo' -Dexec.args='${project.version}' \
                      --non-recursive org.codehaus.mojo:exec-maven-plugin:1.3.1:exec)
   echo CS_version: ${CS_POM_VERSION}
-  checkout_from https://github.com/checkstyle/xwiki-commons.git
+  checkout_from https://github.com/Abhishek-kumar09/xwiki-commons.git
```

Comment:
https://github.com/xwiki/xwiki-commons/pulls/Abhishek-kumar09

please send PR to them.

send PR to our repo - https://github.com/checkstyle/xwiki-commons/tree/5951-indentation

Reply:
Done 
https://github.com/checkstyle/xwiki-commons/pull/1

Reply:
Merged.
Please update to use our fork.

Please send PR to xwiki project to let them quickly merge it after we release your fix.

Reply:
> Please send PR to xwiki project to let them quickly merge it after we release your fix.

I sent it yesterday, and they had merged. 
https://github.com/xwiki/xwiki-commons/pull/117

> Please update to use our fork.

done